Add a dialog when the two Arfectas are destroyed

zimmerwald opened the pull request:
This PR adds a dialog when the two Arfectas are destroyed, to make it clear to the player that they are now an enemy of the Wanderers. It also doesn't make sense that after the two Arfectas are destroyed, the Pug will not replace the two (the Unfettered are still a threat), so four new Arfectas are added to replace the two. Why four? Because of the player.

Commit:
More consequences of destroying the Pug Arfectas

Adds a dialog when the two Arfectas are destroyed (to make it clear to the player that they are now an enemy of the Wanderers), and add four new Arfectas to replace the two.

Commit:
minor fix

removing the extra quotation mark in line 2446

tehhowch commented on data/wanderers start.txt:
```diff
@@ -2443,8 +2443,23 @@ mission "Wanderers Ap'arak 3: Pug"
 		ship "Pug Arfecta" "Hork Ekekek Ptui"
 		ship "Pug Arfecta" "Graak Sput Kchoo"
 	on complete
+		dialog Shortly after destroying the Pug ships, you received a message from Sobari Tele'ek: "We have received [news, reports] that you are destroying the ships of our Keeper [friends, allies] in the Ap'arak system. This is [unacceptable, intolerable]. From now on, you are no longer [welcome, allowed] in our [territory, space]."
```
Needs quoting. Use backticks to wrap the dialog.

tehhowch commented on data/wanderers start.txt:
```diff
@@ -2443,8 +2443,23 @@ mission "Wanderers Ap'arak 3: Pug"
+		dialog Shortly after destroying the Pug ships, you received a message from Sobari Tele'ek: "We have received [news, reports] that you are destroying the ships of our Keeper [friends, allies] in the Ap'arak system. This is [unacceptable, intolerable]. From now on, you are no longer [welcome, allowed] in our [territory, space]."
 		"reputation: Wanderer" <?= -1
 
+mission "Wanderers Ap'arak Pug Patrol Increased"
```
I'm not merging this mission yet, so please remove it

zimmerwald replied:
Removed.

Commit:
Requested changes by tehhowch
Removed the mission "Wanderers Ap'arak Pug Patrol Increased", added backticks in the dialog

tehhowch commented on data/wanderers start.txt:
```diff
@@ -2443,6 +2443,7 @@ mission "Wanderers Ap'arak 3: Pug"
 		ship "Pug Arfecta" "Hork Ekekek Ptui"
 		ship "Pug Arfecta" "Graak Sput Kchoo"
 	on complete
+		dialog `Shortly after destroying the Pug ships, you received a message from Sobari Tele'ek: "We have received [news, reports] that you are destroying the ships of our Keeper [friends, allies] in the Ap'arak system. This is [unacceptable, intolerable]. From now on, you are no longer [welcome, allowed] in our [territory, space]."`
```
Five multiple-meaning blocks in three (almost two) sentences seems high - what's the average use in the missions around it?

I think reducing it to 2 or 3 would be much more palatable. Perhaps `Keeper []` to `Keepers`, `[unacceptable, intolerable]` to `unforgivable`, and `in our [territory, space]` to `here`.

Doesn't have to be those exactly.

zimmerwald replied:
What about this:

"We have received [news, reports] that you are destroying the ships of our Keepers in the Ap'arak system. This is unacceptable. From now on, you are no longer welcome in our [territory, space]."

zimmerwald replied:
I'm gonna commit it then

Commit:
Reduced multiple-meaning blocks to 2
Per https://github.com/tehhowch/endless-sky/pull/29#discussion_r197104299

DarcyManoel commented:
All in for Wanderers becoming hostile on taking the Arfectas. Not too sure about plopping more in there afterwards since the players will likely just steal them too, but you are right that it's weird that the Wanderers just get left without help from the Pug if the player steals the two.
